it/mail/chrome/messenger/accountCreation.dtd

```diff
@@ -60,27 +60,27 @@
 <!ENTITY changeSettings.label            "Cambia impostazioni">
 <!ENTITY changeSettings.accesskey        "m">
 
-<!ENTITY pb.contactYourProvider.description "&brandShortName; can allow you to get to your mail using the provided configurations. However, you should contact your administrator or email provider regarding these improper connections.">
+<!ENTITY pb.contactYourProvider.description "&brandShortName; può consentirti di collegarti alla tua e-mail utilizzando le configurazioni fornite. Tuttavia, dovresti contattare il tuo amministratore o il tuo fornitore di servizi e-mail in merito a queste connessioni non corrette.">
```

Comment:
CORRECTIONS NEEDED. Too many possessive pronouns are redundant. I would suggest to modify the sentence as follows: 

 +<!ENTITY pb.contactYourProvider.description "&brandShortName; può consentire di collegarti alla tua e-mail utilizzando le configurazioni fornite. Tuttavia, dovresti contattare l'amministratore o il fornitore di servizi e-mail in merito a queste connessioni non corrette.">